src/hotspot/os/linux/cgroupV2Subsystem_linux.cpp

```diff
@@ -182,6 +182,16 @@ char* CgroupV2Subsystem::mem_swp_limit_val() {
   return os::strdup(mem_swp_limit_str);
 }
 
+// memory.swap.current : total amount of swap currently used by the cgroup and its descendants
+char* CgroupV2Subsystem::mem_swp_current_val() {
+  GET_CONTAINER_INFO_CPTR(cptr, _unified, "/memory.swap.current",
+                         "Swap currently used is: %s", "%s", mem_swp_current_str, 1024);
```

Comment:
Would love to see 'K' instead of 1024. 'K' is declared in share/utilities/globalDefinitions.hpp
This is just one of multiple occurrences. I do not mark all of them.

Reply:
This could get changed upstream, but please not in a backport!

Reply:
Yes I will look into this upstream, not in the backport !